_includes/setup/android-setup.md

```diff
@@ -0,0 +1,53 @@
+## Android setup
+
+_This step is optional, and can be performed at any time before
+using Flutter to develop apps for Android._
+
+### Install Android Studio
+
+To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit) machine:
```

Comment:
`you can use either a Mac or a Linux (64-bit) machine` is no longer true (it's also windows).

But given all three OS's support Android Studio, just leave out all together